packaging/hoodie-hadoop-mr-bundle/pom.xml

```diff
@@ -211,6 +211,7 @@
                   <include>com.twitter.common:objectsize</include>
                   <include>commons-logging:commons-logging</include>
                   <include>commons-io:commons-io</include>
+                  <include>com.esotericsoftware:kryo-shaded</include>
                 </includes>
```

Comment:
You would need to include the dependencies of kryo-shaded. 
com.esotericsoftware » minlog
& org.objenesis » objenesis 

Please run the demo steps locally http://hudi.incubator.apache.org/docker_demo.html to verify if all the steps succeeds.

Also, you would need similar changes in hoodie-utilities

Reply:
done